### PR TITLE
Add takeBetween signal process feature

### PR DESCRIPTION
**Pull request by pmros**
Useful for drag and drop events, for example.

### CODE REVIEW

File: packages/main/signals/processes/takeBetween.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
+ * `takeBetween` gets the current value in a process and pass it
+ * if and only if it was emitted between two consecutive values
+ * of another process, which acts like a a switch
+ *
+ * It's usage is similar to `merge`:
```
Comment (joaomilho) [inline packages/main/signals/processes/takeBetween.js]: Its

File: packages/main/signals/processes/takeBetween.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * ```
+ * sourceA --x--x----x----x-
+ * sourceB o--o---o---o-o--o
+ * emits   ---o-------o-o---
+ *
+ */
+
+import curry from 'ramda/src/curry'
+
+export default curry((eventSourceSwitch, eventSourceValues, next) => {
```
Comment (joaomilho) [inline packages/main/signals/processes/takeBetween.js]: love the naming

**Comment by joaomilho** [comment]
First thing I'll do is to stabilize tests.

File: packages/main/signals/processes/takeBetween.js (hunk 1 above)
Comment (pmros) [inline packages/main/signals/processes/takeBetween.js]: Oops a typo! Sorry.

Comment (pmros) [inline packages/main/signals/processes/takeBetween.js]: Another typo: 
Line 4: "acts like a a switch" ("a" twice)

**Comment by joaomilho** [comment]
Related to this, I haven't though yet about things that require multiple signals in the view layer. By now you can only "attach" one signal to one event, like in: `['button', {click: ACTION}]`. Added a question on #30

**Pull request merged**